### DISCUSSION TITLE
nginx Error during upload,

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#2894] [Help] [answered]
On an nginx server, when attaching an image, the preview gives me the following error: Error during upload, locally it works perfectly.

![imagen](https://user-images.githubusercontent.com/13526704/175127862-e2c4bf98-906c-4f15-82a6-6648ac32f261.png)

Any suggestions? Plz :)

**Comment by ryangjchandler** [comment]
First thought is your max_upload_size is too low.

**Reply by jcgmalave** [reply]
all values ​​are maxed out :(

**Reply by danharrin** [reply]
Posting the actual error would be useful, if you check the Laravel / console logs

**Reply by jcgmalave** [reply]
the laravel log is empty, and the browser console shows me this: POSThttps://realworld.uscreativity.com/livewire/upload-file?expires=1655933055&signature=4e5405182ea90977f281a653635172f97a09e66f68a30ff5582210254ec35bcc

**Reply by jcgmalave** [reply]
also this: 401
Unauthorized

**Reply by jcgmalave** [reply]
Failed to load resource: the server responded with a status of 401 () https://realworld.uscreativity.com/livewire/upload-file?expires=1655933743&signature=25cdc9b4042a604fbb4652f0a728c09745e352b780ae1762108f32e7c7da769a

**Reply by danharrin** [reply]
https://github.com/livewire/livewire/discussions/3084

**Reply by jcgmalave** [reply]
thanks a lot guys, im gonna keep looking and i will let you know the status. :)

**Reply by jcgmalave** [reply]
Nothing guys; is there a way to display the input file without the livewire preview?

**Reply by danharrin** [reply]
this issue will prevent you from uploading any file, not just through Filepond. its the upload that failed, not the preview

its gonna be something on your configuration that is wrong

**Comment by jcgmalave** [comment, marked as answer]
in case it can help someone, we had to do this to get the functionality of uploading images to an nginx server to work

![imagen](https://user-images.githubusercontent.com/13526704/175564857-dc9142f3-63ca-4939-a9f0-74a488f51161.png)
